create_derivatives/routines.py

```diff
@@ -78,8 +78,27 @@ class ManifestMaker:
 
 
 class AWSUpload:
-    # TO DO: upload files and PDFs
-    pass
+
+    def __init__(self):
+        self.aws_client = AWSClient(*settings.AWS)
+
+    def run(self, replace):
+        uploaded_bags = []
+        for bag in Bag.objects.filter(process_status=Bag.MANIFESTS_CREATED):
+            pdf_dir = str(Path(bag.bag_path, 'data', 'PDF'))
+            jp2_dir = str(Path(bag.bag_path, 'data', 'JP2'))
+            manifest_dir = str(Path(bag.bag_path, 'data', 'MANIFEST'))
```

Comment:
2 things:
- let's either use single quotes or double quotes across the board.
- You can coerce the Path instance to a string once in line 91 or 96, which would save you doing it three times here.